ydb/library/login/login.h

```diff
@@ -171,6 +171,7 @@ class TLoginProvider {
     TBasicResponse ModifyUser(const TModifyUserRequest& request);
     TRemoveUserResponse RemoveUser(const TString& user);
     bool CheckUserExists(const TString& user);
+    bool CheckSidExistsOrIsNonYdb(const TString& sid);
```

Comment:
this library knows nothing about YDB. it would be better to keep this library clean of any YDB specifics or dependencies.

Reply:
OK